lighthouse-core/test/report/html/renderer/report-ui-features-test.js

```diff
@@ -395,5 +395,50 @@ describe('ReportUIFeatures', () => {
         assert.strictEqual(nextNode, nodes[2]);
       });
     });
+
+    describe('onMenuFocusOut', () => {
+      it('should toggle active class when focus relatedTarget is null', () => {
+        dropDown._toggleEl.click();
```

Comment:
nit: move these first two lines to `beforeEach` scoped to this new `describe`?

Reply:
Good idea, thanks! Done.

Reply:
Good idea, thanks! Done.